radiosity/slowrad.js

```diff
@@ -171,4 +177,22 @@ export default class SlowRad {
 
     return this;
   }
+
+  * prepGenerator() {
```

Comment:
Good thinking for using an Iterator to keep track of the form factor and distances